CRM/Utils/System.php

```diff
@@ -1416,7 +1416,7 @@ public static function flushCache() {
     CRM_Contact_BAO_Contact::$_importableFields = CRM_Contact_BAO_Contact::$_exportableFields
       = CRM_Contribute_BAO_Contribution::$_importableFields
         = CRM_Contribute_BAO_Contribution::$_exportableFields
-          = CRM_Pledge_BAO_Pledge::$_exportableFields = CRM_Contribute_BAO_Query::$_contributionFields
```

Comment:
now in Civi::statics so otherwise purged. Not on it's own line so no point for me